src/pudl/package_data/eia176/table_file_map.csv

```diff
@@ -0,0 +1,2 @@
+table,filename
+company,all_company_176.csv
```

Comment:
There will be more tables to add here once we get the simple piece for `company` going.